### ISSUE TITLE
Error: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by ztytotoro** [OfficeDev/office-ui-fabric-react#11051]
### Environment Information

- **Package version(s)**: 7.56.2
- **Browser and OS versions**: 

#### Actual behavior:
```log
import './version';
^^^^^^
SyntaxError: Cannot use import statement outside a module
    at Module._compile (internal/modules/cjs/loader.js:892:18)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (D:\Repos\Local\Blog.Web\node_modules\office-ui-fabric-react\lib-commonjs\components\FloatingPicker\Suggestions\SuggestionsControl.js:8:17)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (D:\Repos\Local\Blog.Web\node_modules\office-ui-fabric-react\lib-commonjs\components\FloatingPicker\BaseFloatingPicker.js:8:28)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
```
#### Expected behavior:
It can work with the version 7.52.0

I compared the difference between new and old `office-ui-fabric-react\lib-commonjs\components\FloatingPicker\BaseFloatingPicker.js`

There is a line newly added:

```js
var Styling_1 = require("office-ui-fabric-react/lib/Styling");    // <--- caused error
```

Maybe you shouldn't import es modules in a commonjs module?

**Comment by maxwellred** [comment]
Hi @ztytotoro Since you've updated the repo, I'm wondering if perhaps you need to re-install deps with `yarn` and re-build the project with `yarn builddemo` OR `yarn build` ?

**Comment by ztytotoro** [comment]
> Hi @ztytotoro Since you've pulled down the latest repo version, I'm wondering if perhaps you need to re-install deps with `yarn` and then re-build the project with `yarn builddemo` OR `yarn build` ?

Thanks for your reply!

I pulled down this repo just want to figure out where the import used and why, I'm not sure of solving this problem my self. 

Actually this problem appears in my `next.js` app, I upgraded the packages recently and then it broked. You can find the source code [here](https://github.com/ztytotoro/Blog.Web)

**Comment by maxwellred** [comment]
Hi @ztytotoro I have checked with colleagues and received this [workaround](https://github.com/OfficeDev/office-ui-fabric-react/issues/10950#issuecomment-550075330) you can apply to your jest.config.js. Give it a try and let me know, thanks!

**Comment by ecraig12345** [comment]
Actually I think we should remove that root-level import (and do a relative import instead)—we shouldn’t have root-level imports in component files for exactly this reason. 

In addition, I recently made an issue about adding a lint-imports rule preventing more imports like this from being added. I could potentially just go ahead and add that, or walk @maxwellred through what to do if you have time (it shouldn’t be very hard). 
https://github.com/OfficeDev/office-ui-fabric-react/issues/10989

**Comment by cliffkoh** [comment]
Weird, I don't see 
```ts
var Styling_1 = require("office-ui-fabric-react/lib/Styling");    // <--- caused error
```

at https://unpkg.com/browse/office-ui-fabric-react@7.56.2/lib-commonjs/components/FloatingPicker/BaseFloatingPicker.js

**Comment by ecraig12345** [comment]
You're right, I don't see an import like that either in the source for that file (and the file hasn't been modified recently).
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.tsx

I think the issue is actually in SuggestionsControl, which *was* modified more recently:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/FloatingPicker/Suggestions/SuggestionsControl.tsx

**Comment by ztytotoro** [comment]
> Hi @ztytotoro I have checked with colleagues and received this [workaround](https://github.com/OfficeDev/office-ui-fabric-react/issues/10950#issuecomment-550075330) you can apply to your jest.config.js. Give it a try and let me know, thanks!

Thanks, I didn't run any test just cannot start my react app. I checked the error info, as @ecraig12345 said, the code caused error is in:
```sh
node_modules\office-ui-fabric-react\lib-commonjs\components\FloatingPicker\Suggestions\SuggestionsControl.js
```
I think the way of importing an es module in commonjs file makes the pack system broken. Maybe you should import it from the root `office-ui-fabric-react` instead of `office-ui-fabric-react/lib/Styling`, or use a relative path, thanks for your efforts again.

**Comment by ztytotoro** [comment]
Problem seems solved in the latest release 7.59.3, I will close this issue
